core/src/main/java/tc/oc/pgm/score/ScoreMatchModule.java

```diff
@@ -50,6 +49,8 @@
 
   private final Match match;
   private final ScoreConfig config;
+  private final int defaultScore;
+  private int scoreLimit;
```

Comment:
Move the mutable part of the state lower, don't mix in immutable and mutable properties.

The immutable state in this class goes up to the Set of scorebox, contributions and forward are mutable state